Build/set-dependencies.sh

```diff
@@ -116,5 +117,7 @@ done
 
 php "${COMPOSER_PHAR}" --working-dir=Packages/Framework/Neos.Kickstarter require --no-update "neos/fluid-adaptor:~${BRANCH}.0"
 php "${COMPOSER_PHAR}" --working-dir=Packages/Application/Neos.Welcome require --no-update "neos/flow:~${BRANCH}.0"
+php "${COMPOSER_PHAR}" --working-dir=Packages/Application/Neos.Behat require --no-update "neos/flow:~${BRANCH}.0"
 
 commit_manifest_update ${BRANCH} "${BUILD_URL}" ${VERSION} "Packages/Framework"
+commit_manifest_update ${BRANCH} "${BUILD_URL}" ${VERSION} "Packages/Application"
```

Comment:
This line most surely was the reason why #46 occured with missing Flow version in the welcome composer

Reply:
But… there is no `Application/composer.lock`, eh? What and where would that commit?

Reply:
Ah 🙈 right, that should be `Packages/Application/Neos.Behat` and `Packages/Appliaction/Neos.Welcome`
```suggestion
commit_manifest_update ${BRANCH} "${BUILD_URL}" ${VERSION} "Packages/Application/Neos.Behat"
commit_manifest_update ${BRANCH} "${BUILD_URL}" ${VERSION} "Packages/Application/Neos.Welcome"
```

Reply:
Yep. As it says, it commits an update to a manifest. ;)